amida-tech/indaba-client
# Feature/INBA-456 Connect unarchive message to backend

#### What does this PR do?
Use the new `/unarchive` endpoint of the messaging service to unarchive messages instead of mocking the call.

#### Related JIRA tickets:
[INBA-456](https://jira.amida-tech.com/browse/INBA-456)

#### How should this be manually tested?
1. Log in as a user who has messages
1. Archive a thread from the inbox
1. Go to the archived messages
1. Unarchive a thread
1. Check that the thread is immediately returned to the inbox _and_ persists in the inbox after a refresh

#### Background/Context

#### Screenshots (if appropriate):
